playbooks/provision_openstack_instance.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: CVP 1164
+  hosts: '{{ hosts | default("all") }}'
```

Comment:
You can default the hosts to localhost instead of all

Reply:
The playbook does not run when the default is localhost instead of all.